genestack_client/settings/genestack_config.py

```diff
@@ -127,6 +137,11 @@ def change_password(self, alias, password):
         user.password = password
         self.save()
 
+    def change_token(self, alias, token):
```

Comment:
I really think we need consistency on that "set/change token" question.

Reply:
Let it change.